authors/apps/articles/migrations/0002_article_tags.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2 on 2019-04-09 15:04
+
+import django.contrib.postgres.fields
+from django.db import migrations, models
+
```

Comment:
@ezrogha kindly delete all migrations and have only one migration file

Reply:
Thanks @dannylwe, 
I have worked on it